int16 button0
int16 button1
int16 button2
